openreview/venue_request/process/matchingProcess.py

```diff
@@ -23,62 +23,62 @@ def process(client, note, invitation):
         matching_status = conference.setup_committee_matching(matching_group, compute_affinity_scores, compute_conflicts)
     except Exception as e:
         if 'Submissions not found.' in str(e):
-            matching_status['error'] = ['Could not compute affinity scores and conflicts since no submissions were found. Make sure the submission deadline has passed and you have started the review stage using the \'Review Stage\' button.']
+            matching_status['error'] = 'Could not compute affinity scores and conflicts since no submissions were found. Make sure the submission deadline has passed and you have started the review stage using the \'Review Stage\' button.'
         elif 'The match group is empty' in str(e):
-            matching_status['error'] = [f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.']
+            matching_status['error'] = f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.'
         elif 'The alternate match group is empty' in str(e):
             role_name = conference.get_area_chairs_name()
-            matching_status['error'] = [f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.']
+            matching_status['error'] = f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.'
         else:
-            matching_status['error'] = [str(e)]
+            matching_status['error'] = str(e)
 
     print("Following error in the process function was posted as a comment:")
     print(traceback.format_exc())
 
     comment_note = openreview.Note(
-        invitation = note.invitation.replace('Paper_Matching_Setup', 'Comment'),
+        invitation = note.invitation.replace('Paper_Matching_Setup', 'Paper_Matching_Setup_Status'),
```

Comment:
All the existing venues must have this invitation created so we need to keep backward compatibility if we don't want to create these invitations for previous venues. 

I suggest to check if the Paper_Matching_Setup_Status invitation exists then use it otherwise keep using the comment invitation.

Reply:
It's best to make invitations for the existing venues. If we switch to `Comment` invitation, we will have to make a different new note which matches the `Comment` invitation structure.

Reply:
Ok, please remember to prepare the script to create the invitations for all the venues before we deploy this change in the live site.